lib/nel.js

```diff
@@ -51,7 +51,7 @@ var spawn = require("child_process").spawn;
 
 var transform = require("babel-core").transform;
 var transformOptions = {
-    presets: ["es2015"],
+    presets: [path.resolve(path.join(__dirname, "..", "node_modules", "babel-preset-es2015"))],
```

Comment:
Nice! I didn't realise that `presets` could take paths like that.

I think the following would be more robust:
```js
var transformOptions = {
    presets: [require.resolve("babel-preset-es2015")],
};
```